docs/data/material/pages.ts

```diff
@@ -184,6 +184,7 @@ const pages: MuiPage[] = [
           { pathname: '/material-ui/customization/typography' },
           { pathname: '/material-ui/customization/spacing' },
           { pathname: '/material-ui/customization/breakpoints' },
+          { pathname: '/material-ui/customization/containerQueries', newFeature: true },
```

Comment:
```suggestion
          { pathname: '/material-ui/customization/containerQueries', newFeature: true, title: "Container queries" },
```

As it is currently rendering as "ContainerQueries"